libraries/chain/controller.cpp

```diff
@@ -1203,7 +1203,7 @@ struct controller_impl {
    }
 
    int64_t remove_scheduled_transaction( const generated_transaction_object& gto ) {
-      if (auto dm_logger = get_deep_mind_logger()) {
+      if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
`remove_scheduled_transaction` is not called for read-only or dry-run transactions.

Reply:
Please add a comment that says this is only called from `push_scheduled_transaction`